gameoflife.py

```diff
@@ -19,51 +19,77 @@
                                                               
                                                               '''
 
-def init_board(file_path):
-    b = []
-    if not os.path.isfile(file_path):
-        raise Exception(f"File {file_path} could not be found.")
-    with open(file_path, mode="r") as file:
-        file_content = file.readlines()
+description = '''Game of life is a zero-player game, which means it is detemined by it's initial state. You get to decide what the board looks like at the start of the game, using a text file according to the format (see file_format.txt).
+Every square on the board represents a cell, which can be dead or alive. In each generation a cell can die, revive or stay the same according to it's neighbours(the 6 surrounding cells).
+The rules are:
+\t[1] Any live cell with two or three live neighbours survives.
+\t[2] Any dead cell with three live neighbours becomes a live cell.
+\t[3] All other live cells die in the next generation, and all other dead cells stay dead.
+'''
```

Comment:
Add a README.md file and move this description there (you can read about .md files in the github docs)